app/Console/Commands/UserCreate.php

```diff
@@ -71,23 +87,52 @@ public function handle(DiscourseService $discourseService): void
         {
-            $user = User::create([
-                                     'name' => $name,
-                                     'email' => $email,
-                                     'password' => Hash::make($password),
-                                     'role' => Role::RESTARTER,
-                                     'recovery' => substr(bin2hex(openssl_random_pseudo_bytes(32)), 0, 24),
-                                     'recovery_expires' => strftime('%Y-%m-%d %X', time() + (24 * 60 * 60)),
-                                     'calendar_hash' => Str::random(15),
-                                     'username' => '',
-                                     'wiki_sync_status' => WikiSyncStatus::CreateAtLogin,
-                                     'language' => $language,
-                                     'repair_network' => $repair_network_id,
-                                 ]);
+            $userData = [
+                'name' => $name,
+                'email' => $email,
+                'password' => Hash::make($password),
+                'role' => $role,
+                'recovery' => substr(bin2hex(openssl_random_pseudo_bytes(32)), 0, 24),
+                'recovery_expires' => strftime('%Y-%m-%d %X', time() + (24 * 60 * 60)),
+                'calendar_hash' => Str::random(15),
+                'username' => '',
+                'wiki_sync_status' => WikiSyncStatus::CreateAtLogin,
+                'language' => $language,
+                'repair_network' => $repair_network_id,
+            ];
+
+            // Add consent data if provided
+            if (!empty($consent_past_data)) {
+                $userData['consent_past_data'] = $consent_past_data;
+            }
+            
+            if (!empty($consent_future_data)) {
+                $userData['consent_future_data'] = $consent_future_data;
+            }
+            
+            if (!empty($consent_gdpr)) {
+                $userData['consent_gdpr'] = $consent_gdpr;
+            }
+
+            $user = User::create($userData);
 
             if ($user)
             {
                 $this->info("User created #" . $user->id);
+                $this->info("Role: " . $this->getRoleName($user->role));
+                
+                if (!empty($userData['consent_past_data']) || !empty($userData['consent_future_data']) || !empty($userData['consent_gdpr'])) {
+                    $this->info("Consent dates set:");
+                    if (!empty($userData['consent_past_data'])) {
+                        $this->info("- Past data: {$userData['consent_past_data']}");
+                    }
+                    if (!empty($userData['consent_future_data'])) {
+                        $this->info("- Future data: {$userData['consent_future_data']}");
+                    }
+                    if (!empty($userData['consent_gdpr'])) {
+                        $this->info("- GDPR: {$userData['consent_gdpr']}");
+                    }
+                }
```

Comment:
```suggestion
                if ($userData['consent_past_data'] || $userData['consent_future_data'] || $userData['consent_gdpr']) {
                    $this->info("Consent dates set:");
                    if ($userData['consent_past_data']) {
                        $this->info("- Past data: {$userData['consent_past_data']}");
                    }
                    if ($userData['consent_future_data']) {
                        $this->info("- Future data: {$userData['consent_future_data']}");
                    }
                    if ($userData['consent_gdpr']) {
                        $this->info("- GDPR: {$userData['consent_gdpr']}");
                    }
                }
```

Reply:
Addressed in [c428255](https://github.com/iFixit/restarters/pull/15/commits/c4282554dc2afebfc3aa1dab2415c40d36480569)

I ended up just dropping the logs. They were more beneficial in the beginning but now not anymore.